calculator-app/src/main/java/camp/nextstep/edu/calculator/CalculatorViewModelProviderFactory.kt

```diff
@@ -0,0 +1,17 @@
+package camp.nextstep.edu.calculator
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import camp.nextstep.edu.calculator.domain.RecordRepository
+
+class CalculatorViewModelProviderFactory(
```

Comment:
```suggestion
class CalculatorViewModelFactory(
```

이름에는 이름+ViewModelFactory만 되어도 의미 전달을 분명히 할 수 있습니다!